budproj/routines-microservice
# feat(routine-settings)

## 🎢 Motivation

Allow admins to configure companies' routines

## 🔧 Solution

Create a route to save routine settings and schedule notifications

## 🚨  Testing

- Execute this micro-service
- Execute the schedule micro-services
- Make a request for the /settings route
- Check if the settings was stored in the database
- Check if the notifications schedule was stored to the schedule database

## 🔗 Related PRs

This PR is related to some other PRs in different services, they are:

- [`scheduler-microservice#13`](https://github.com/budproj/scheduler-microservice/pull/13)
- [`business#213`](https://github.com/budproj/business/pull/213)

## 🍩 Validation

Who tested this PR?

### Canary


### Dev
